R/espn_league.R

```diff
@@ -27,6 +27,7 @@ ff_league.espn_conn <- function(conn) {
   tibble::tibble(
     league_id = conn$league_id,
     league_name = league_endpoint$content$settings$name,
+    season = conn$season,
```

Comment:
Looks great so far! Can you make these integers across the board for consistency?

Reply:
all set